build/luarocks/BUILD.luarocks.bazel

```diff
@@ -160,6 +169,9 @@ genrule(
         # alias LDOC command to true(1) command
         export LDOC=true
 
+        # Install lua-messagepack
+        $(location :luarocks_exec) install $(location :prepare_lua_messagepack) 2>&1 >$@.tmp
+
         $(location :luarocks_exec) make --no-doc 2>&1 >$@.tmp
```

Comment:
I think both of these commands output needs to be appended `>>` instead of redirected `>` since there are two commands now, no?

Reply:
Also the sequence order of the `2>&1` and `>$@.tmp` seems weird, usually the order should be reversed(if you want both stdout/stderr recorded in the file)
Not sure what is the original intention of this...

Reply:
done